testsuite/features/secondary/trad_migrate_to_minion.feature

```diff
@@ -112,3 +116,7 @@ Feature: Migrate a traditional client into a Salt minion
     Given I am on the Systems overview page of this "sle_client"
     When I follow "Properties" in the content area
     Then I wait until I see "Base System Type:.*Management" regex, refreshing the page
+
+  Scenario: Cleanup: check that we still have the activation key
```

Comment:
In my opinion this is not a real cleanup and therefore the text `Cleanup` can be omitted.

Reply:
For future reference, after discussion, it was decided we'll leave it as it is, since it's confirming the cleanup and therefore is a part of it

Reply:
I used `[this](https://github.com/uyuni-project/uyuni/blob/master/testsuite/documentation/guidelines.md#rules-for-scenarios)` for reference. But as Eric suggested, it is still a cleanup. So resolved.